src/domain/metric/service/postMetric.js

```diff
@@ -30,7 +31,7 @@ export default (posts: Post[]): PostMetric[] => {
     return {
       post,
       id: post.id,
-      createdTime: new Date(post.created_time),
+      createdTime: moment(post.created_time).format(),
```

Comment:
@reyhansofian yeah, it's not a problem btw. The problem is the `posts` count is always 0 on iOS (even on Chrome). The `date invalid` is not a problem. We parse it right on the Component, like this https://github.com/hellowin/kanca/blob/master/src/domain/metric/component/Pie.js#L134

![image](https://cloud.githubusercontent.com/assets/1904744/25778861/cc2169c6-3334-11e7-9391-37e931f4a6c8.png)

As you can see, the problem I mentioned is not the `invalid date` but the non-existence of the data itself.